src/issues/issue-2/david-byrne/index.js

```diff
@@ -0,0 +1,161 @@
+import React, { useState, useEffect } from "react"
+import Img from "gatsby-image"
+import { useStaticQuery, graphql } from "gatsby"
+import { useMediaQuery } from "react-responsive"
+
+import Header from "./Header"
+import AnimatedWords from "./AnimatedWords"
+import Pagination from "./Pagination"
+
+import useMedia from "./useMedia"
+import useMeasure from "./useMeasure"
+import usePositions from "./usePositions"
+import albumByYearData from "./albumsByYear.js"
+import { mapImagesToAlbums, calculateGridItems } from "./utility"
+import {
+  Background,
+  Text,
+  FlexContainer,
+  ImgWrap,
+  WordCountContainer,
+} from "./styled"
+import { breakpoints } from "../../../styles/layout"
+
+const xOffset = window.innerWidth / 2
+
+const DavidByrne = () => {
+  const isPhoneWide = useMediaQuery({
+    query: breakpoints.phoneWide,
+  })
+  // Query images
+  const { mini, big } = useStaticQuery(graphql`
+    query {
+      mini: allFile(
```

Comment:
I realized you can target multiple folders with graphql by assigning them to properties like `mini` and then destructuring them

Reply:
can we go over what mini is?